<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiCameras</class>
 <widget class="QDialog" name="MultiCameras">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Multi Camera Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolTip{
	color : black;
}
QDialog{
	background : #333333;
}
QGroupBox{
	color : white;
	margin-top : 2ex;
	border-radius : 5px;
	padding : 8px;
	padding-top : 13px;
	 border-style: outset;
 	 border-width: 2px;
	 border-color: #1f1f1f;
}
QMenuBar#menuBar{

	background : grey;
	color : white;
	 border-bottom-style: outset;
 	 border-bottom-width: 1px;
	 border-bottom-color: #222222;
}
QMenu{
	background : grey;
	color : #1f1f1f;
}
QMenu::item:selected{
	background : #c9c9c9;
}
QStatusBar#statusBar{
	background : grey;
	color : white;
	border-top-style: outset;
 	border-top-width: 1px;
	border-top-color: #222222;
}
QPushButton{
	color : white;
	background : grey;
	padding : 5px;
	border-radius : 3px;
}
QPushButton:pressed{
	background : #474747;
}
QPushButton:hover{
	background : #c9c9c9;
}
QPushButton:disabled{
	color : #474747;
}
QLineEdit{
	color : white;
	background : #111111;
	padding : 3px;
	border-radius : 3px;
	border-color : white;
	border-width : 1px;
}
QComboBox{
	color : white;
	background : #111111;
	padding : 3px;
	min-width : 40px;
	border-radius : 3px;

}
QComboBox::drop-down{
	
}
QComboBox QAbstractItemView{
	background-color : black;
	color : white;
}
QSlider::handle{
	background : grey;
}
QSlider::handle:hover{
	background : #c9c9c9;
}
QSlider::handle:pressed{
	background : #474747;
}
QLabel{
	color : white;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Camera 1:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="camIP1">
         <property name="placeholderText">
          <string>IP Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="camName1">
         <property name="placeholderText">
          <string>Camera Name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Camera 2:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="camIP2">
         <property name="placeholderText">
          <string>IP Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="camName2">
         <property name="placeholderText">
          <string>Camera Name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Camera 3:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="camIP3">
         <property name="placeholderText">
          <string>IP Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="camName3">
         <property name="placeholderText">
          <string>Camera Name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Camera 4:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="camIP4">
         <property name="placeholderText">
          <string>IP Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="camName4">
         <property name="placeholderText">
          <string>Camera Name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Camera 5:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="camIP5">
         <property name="placeholderText">
          <string>IP Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="camName5">
         <property name="placeholderText">
          <string>Camera Name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Camera 6:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="camIP6">
         <property name="placeholderText">
          <string>IP Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="camName6">
         <property name="placeholderText">
          <string>Camera Name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Camera 7:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="camIP7">
         <property name="placeholderText">
          <string>IP Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="camName7">
         <property name="placeholderText">
          <string>Camera Name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Camera 8:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="camIP8">
         <property name="placeholderText">
          <string>IP Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="camName8">
         <property name="placeholderText">
          <string>Camera Name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="multiDlgCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="multiDlgOK">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
